code/__DEFINES/configuration.dm

```diff
@@ -37,3 +37,4 @@
 #define REVENANT_NAME_FILE "revenant_names.json"
 #define ITALIAN_TALK_FILE "italian_replacement.json"
 #define VALENTINE_FILE "valentines.json"
+#define GIMMICK_OBJ_FILE "strings/gimmick_objectives.txt"
```

Comment:
this should excluse the strings/ part, strings/ should be added on like the other defines.

Reply:
fixed